### PR TITLE
buffs ripley mining equipment

### PR DESCRIPTION
**Pull request by Iamgoofball**
done in separate PR in case people want just this change instead of the others
1. increased range on the drill
2. halved the cooldown on the drill
3. doubled the speed of the ore pulse

:cl:
tweak: Cooldown on the Ripley's mining drills has been halved.
tweak: The frequency of the Ripley's ore pulse has been doubled.
/:cl:

### CODE REVIEW

File: code/game/mecha/equipment/tools/mining_tools.dm [hunk 1]
```diff
@@ -49,7 +49,7 @@
 		drill.occupant_message("<span class='danger'>[src] is too durable to drill through.</span>")
 
 /turf/closed/mineral/drill_act(obj/item/mecha_parts/mecha_equipment/drill/drill)
-	for(var/turf/closed/mineral/M in range(drill.chassis,1))
+	for(var/turf/closed/mineral/M in range(drill.chassis,3))
```
Comment (ChangelingRain) [inline code/game/mecha/equipment/tools/mining_tools.dm]: this'll create a weird cone and the speed buff might be more than enough

**Comment by GunHog** [comment]
The drills were hilariously inferior for mining, and were instead more useful for gibbing goliaths for the hides, and watchers for the diamond. A good change.

**Comment by Bawhoppen** [comment]
The issue isn't that the drill is underpowered, but that everything else is overpowered

File: code/game/mecha/equipment/tools/mining_tools.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/game/mecha/equipment/tools/mining_tools.dm]: i think it'll be fine

**Review by imtakingabreakdontatme — DISMISSED** [review]
Might need to tweak what Joan said but overall a good PR

File: code/game/mecha/equipment/tools/mining_tools.dm (hunk 1 above)
Comment (ChangelingRain) [reply code/game/mecha/equipment/tools/mining_tools.dm]: i suggest getting a picture.

**Comment by Buggy123** [comment]
Merge posthaste, I want mining Ripleys to be useful again.

**Comment by KorPhaeron** [comment]
Mining ripleys will never be good until rnd material costs are high enough to warrant strip mining

**Comment by AnonymousNow** [comment]
See, Goof? You can make PRs that are good, popular, and good & popular sometimes.

**Review by ChangelingRain — CHANGES_REQUESTED** [review]
get me a picture of the drill cone, goof, a picture!

**Comment by Bawhoppen** [comment]
@AnonymousNow Only reason it's popular is because something is being buffed. A lot of people are babies and can't handle anything being nerfed.

**Comment by KorPhaeron** [comment]
@ChangelingRain I fixed what you wanted

**Comment by Iamgoofball** [comment]
eh i'll do the cone later

**Comment by PopNotes** [comment]
remind me again why people think this is okay
the ripley is already by far and large the best way of mining short of bombs, it doesn't need to be made even better

(it also still fucking gibs corpses for whatever reason, and now at double the speed)

**Comment by AnonymousNow** [comment]
Uh, no, by far and away the best way of mining is using a plasma cutter. Which humans can do more efficiently.

**Comment by Bawhoppen** [comment]
Have you used a Ripley recently?

**Comment by jughu** [comment]
Being in a ripley is like havin a second healthbar

**Comment by AnonymousNow** [comment]
What, me?

Yes. They're big, slow, power-hungry bastards that're best used for tanking brute damage in the early stages. They get better when time and resources are invested into them - goliath plates, better battery, a mounted cutter - but I don't see that as a problem.

**Pull request merged**